[2023-04-04T10:18:44.540+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: user_processing.store_user manual__2023-04-04T10:11:09.970918+00:00 [queued]>
[2023-04-04T10:18:44.546+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: user_processing.store_user manual__2023-04-04T10:11:09.970918+00:00 [queued]>
[2023-04-04T10:18:44.547+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-04T10:18:44.547+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-04-04T10:18:44.547+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-04T10:18:44.558+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): store_user> on 2023-04-04 10:11:09.970918+00:00
[2023-04-04T10:18:44.562+0000] {standard_task_runner.py:55} INFO - Started process 96087 to run task
[2023-04-04T10:18:44.565+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'store_user', 'manual__2023-04-04T10:11:09.970918+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp4f_woxrc']
[2023-04-04T10:18:44.565+0000] {standard_task_runner.py:83} INFO - Job 33: Subtask store_user
[2023-04-04T10:18:44.609+0000] {task_command.py:389} INFO - Running <TaskInstance: user_processing.store_user manual__2023-04-04T10:11:09.970918+00:00 [running]> on host 1f05588790e0
[2023-04-04T10:18:44.657+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=store_user
AIRFLOW_CTX_EXECUTION_DATE=2023-04-04T10:11:09.970918+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-04T10:11:09.970918+00:00
[2023-04-04T10:18:44.659+0000] {postgres.py:158} INFO - Running copy expert: COPY users FROM stdin WITH DELIMITER as ',', filename: /tmp/processed_user.csv
[2023-04-04T10:18:44.667+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-04-04T10:18:44.671+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-04-04T10:18:44.678+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=store_user, execution_date=20230404T101109, start_date=20230404T101844, end_date=20230404T101844
[2023-04-04T10:18:44.696+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-04T10:18:44.711+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
